src/stylesheets/Book.css

```diff
@@ -0,0 +1,171 @@
+:root {
+  --medium-blue: #307bbe;
+  --dodger-blue: #379cf6;
+  --azure: #0290ff;
+  --cool-blue: #4386bf;
+}
+
+.book-panel {
+  display: flex;
+  width: 80%;
+  height: 170px;
+  margin: 37px 100px 0;
+  padding: 32px 147px 26px 27px;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.book-info {
+  display: flex;
+  flex-direction: column;
+  width: 40%;
+}
+
+.book-category,
+.book-title,
+.percentage,
+.chapter {
+  color: #121212;
+}
+
+.book-category,
+.percentage,
+.completed {
+  font-family: 'Montserrat', sans-serif;
+}
+
+.book-category,
+.percentage,
+.completed,
+.actions span,
+.actions a {
```

Comment:
- [ ] Styling should be EXACTLY the same as the one provided in the designs. Please see the major differences between the two designs below"

Your design:
![Screenshot 2021-09-14 at 13-21-21 React App](https://user-images.githubusercontent.com/70150423/133265320-c6af36c0-1c2a-4e15-b35a-75c3d9e56a4e.png)

Design Template
![Screenshot_2021-09-14_13-25-29](https://user-images.githubusercontent.com/70150423/133266152-3bf30b29-35c8-4451-a771-8de36916f968.png)

Your design does not have the header. The filter feature you implemented in the application is also missing in your design. Please let's fix it.